recipes/ebcdic/meta.yaml

```
@@ -0,0 +1,46 @@
+{% set name = "ebcdic" %}
```

What's the reason for changing the package name?
```suggestion
{% set name = "codecmapper" %}
```

---

Ah I see it's the PyPI name!